src/main/java/com/mraof/minestuck/entity/consort/ConsortEntity.java

```diff
@@ -450,6 +477,25 @@ public static boolean canConsortSpawnOn(EntityType<ConsortEntity> entityType, Le
 		return true;
 	}
 	
+	@Override
+	public Dialogue getDialogue()
+	{
+		if(dialogue != null)
+			return dialogue;
+		else
+			dialogue = getDialogueFromEntity(this);
```

Comment:
```suggestion
		if(dialogue != null)
			return dialogue;
		
		dialogue = getDialogueFromEntity(this);
```

The `else` makes it look like it only might run with the code outside and after the `else`, when it in fact will always run with that code.

Reply:
```suggestion
		if(dialogue == null)
			dialogue = getDialogueFromEntity(this);
```
This is another way of doing it, which would make it match the if-statement below. I'm not entirely sure what to prefer.